pageserver/src/import_datadir.rs

```diff
@@ -331,6 +333,8 @@ pub fn import_basebackup_from_tar<R: Repository, Reader: Read>(
     let _pg_control = pg_control.context("pg_control file not found")?;
 
     modification.commit()?;
+    tline.tline.checkpoint(crate::CheckpointConfig::Flush)?;
```

Comment:
We already call the checkpoint outside of this function in `handle_import_basebackup()`